packages/ripple-nuxt-tide/modules/alert/alert-middleware.js

```diff
@@ -0,0 +1,17 @@
+import { get } from 'lodash'
```

Comment:
So this is actually refresh site data, not only alert. I would like to move this to site module, how do you think?